src/kinematics_plugin.cpp

```diff
@@ -106,17 +106,9 @@ toBioIKKinematicsQueryOptions(const void *ptr) {
 
 namespace bio_ik_kinematics_plugin {
 
-// Fallback for older MoveIt versions which don't support lookupParam yet
-template <class T>
-static void lookupParam(const std::string &param, T &val,
-                        const T &default_val) {
-  ros::NodeHandle nodeHandle("~");
-  val = nodeHandle.param(param, default_val);
-}
-
 struct BioIKKinematicsPlugin : kinematics::KinematicsBase {
+  rclcpp::Node::SharedPtr node_;
```

Comment:
Remove the trailing underscore to match existing style and because that convention isn't used for public data members.

Reply:
Now realizing `node_` is actually a protected member of `KinematicsBase` -- I've removed the declaration here